eclass/perl-module.eclass

```diff
@@ -254,7 +254,7 @@ perl-module_src_configure() {
 	elif [[ -f Makefile.PL ]] ; then
 		einfo "Using ExtUtils::MakeMaker"
 		set -- \
-			PREFIX=${EPREFIX}/usr \
```

Comment:
I have some double slash fix for one of the Perl eclasses, but I'd need to dig it out. Maybe we'll merge that at the same time